docs/resources/samples/automate-tasks-on-all-excel-files-in-folder.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Run a script on all Excel files in a folder'
+description: 'Learn how to run a script on all the Excel files in a folder on OneDrive for Business.'
+ms.date: 03/31/2021
+localization_priority: Normal
+---
+
+# Run a script on all Excel files in a folder
+
+This project performs a set of automation tasks on all files situated in a folder on OneDrive for Business (can also be used for SharePoint folder).
+It performs calculations on the Excel files, adds formatting, and inserts a comment that [@mentions](https://support.microsoft.com/office/90701709-5dc1-41c7-aa48-b01d4a46e8c7) a colleague.
+
+## Office Scripts sample code: Add formatting and insert comment
+
+Download the file <a href="https://github.com/OfficeDev/office-scripts-docs/blob/elizs-transfer-samples/docs/resources/samples/highlight-alert-excel-files.zip?raw=true">highlight-alert-excel-files.zip</a>, extract the files to a folder titled **Sales** used in this sample, and try it out yourself!
```

Comment:
This is pointing to your branch.

Reply:
Yes, so that I can verify this works. I have a note to update to prod version before publishing.